ROADMAP.md

```diff
@@ -1,28 +1,27 @@
 # Roadmap
 
 This document defines a high level roadmap for Rook development and upcoming releases.
-The features and themes included in each milestone are optimistic in the sense that many do not have clear owners yet.
+The features and themes included in each milestone are optimistic in the sense that some do not have clear owners yet.
 Community and contributor involvement is vital for successfully implementing all desired items for each release.
 We hope that the items listed below will inspire further engagement from the community to keep Rook progressing and shipping exciting and valuable features.
 
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
-We use the [milestone](https://github.com/rook/rook/milestones) feature in Github so look there for the most up-to-date and issue plan.
+We use the [project boards](https://github.com/rook/rook/projects) in Github so look there for the most up-to-date issues and their status.
 
 
-## Rook 1.4
+## Rook 1.5
 
-The following high level features are targeted for Rook v1.4 (July 2020). For more detailed project tracking see the [v1.4 board](https://github.com/rook/rook/projects/18).
+The following high level features are targeted for Rook v1.5 (November 2020). For more detailed project tracking see the [v1.5 board](https://github.com/rook/rook/projects/19).
 
 * Ceph
-  * Admission controller [#4819](https://github.com/rook/rook/issues/4819)
-  * RGW Multi-site replication (experimental) [#1584](https://github.com/rook/rook/issues/1584)
-  * Handle IPv4/IPv6 dual stack configurations [#3850](https://github.com/rook/rook/issues/3850)
-  * Support for provisioning OSDs with drive groups [#4916](https://github.com/rook/rook/pull/4916)
+  * RBD mirroring configuration between clusters to support application DR (disaster recovery)
+  * Support for an external KMS (e.g. Vault) for storing OSD encryption keys [#6105](https://github.com/rook/rook/issues/6105)
+  * Support for stretched clusters with an arbiter mon [#5592](https://github.com/rook/rook/issues/5592)
+  * Encryption configuration per pool or per volume via the CSI driver
+  * RGW Multi-site replication improvements towards declaring the feature stable
   * Multus networking configuration declared stable
-  * RBD Mirroring configured with a CRD
-  * All Ceph controllers updated to the controller runtime
-  * Uninstall options for sanitizing OSD devices
-  * Enhancements to external cluster configuration
+  * Handle IPv4/IPv6 dual stack configurations [#3850](https://github.com/rook/rook/issues/3850)
```

Comment:
FYI, dual stack on Kubernetes will be reimplemented in 1.20.
https://github.com/kubernetes/enhancements/issues/563#issuecomment-682316770

Reply:
Thanks for the link. I'm thinking we should instead target to support IPv6 single stack.